express-forms/src/index.js

```diff
@@ -1,19 +1,35 @@
 const express = require('express');
+const path = require('path');
+const bodyParser = require('body-parser');
 
 const app = express();
 
-app.get('/', (req, res) => {
-  return res.redirect('/form-with-get');
+app.use(bodyParser.urlencoded({ extended: true }));
+const publicPath = path.join(__dirname, './views');
```

Comment:
Should you require to define `views` as public path in `express`?
It seems Express make `views` as public path by default.
But it would be good idea if `path name` is different one.